app/Mail/SampleNotification.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SampleNotification extends Mailable
```

Comment:
This is still SampleNotification, you should change to a readable and understandable file name and class name.